packages/core/useDark/index.md

```diff
@@ -37,7 +37,7 @@ By default, it uses [Tailwind CSS favored dark mode](https://tailwindcss.com/doc
 <html class="dark"> ... </html>
 ```
 
-While you can customize it and make it work for most of the CSS frameworks.
+Stillm you can also customize it to make it work with most CSS frameworks.
```

Comment:
Is `Stillm` a typo?

Reply:
Yes, sorry about that. It should read: "Still,"